server/neptune/gateway/event/deploy_workflow_signaler.go

```diff
@@ -48,6 +49,9 @@ func (d *DeployWorkflowSignaler) SignalWithStartWorkflow(
 		workflows.DeployNewRevisionSignalID,
 		workflows.DeployNewRevisionSignalRequest{
 			Revision: revision,
+			User: workflows.User{
```

Comment:
Should this be more specific? What does `User` on this object mean? Something like `InitiatingUser` makes more sense imo.